active/RadioMenuFlyoutItem/RadioMenuFlyoutItem.md

```diff
@@ -0,0 +1,61 @@
+# Description
+A radio button menu flyout item is a menu item that is mutually exclusive with other menu items in its group. 
+This control contol allows apps to present users with menus containing multiple options, where a user would choose only of these options at a given time.
+
+
+# Examples
+
+Add two or more *RadioMenuFlyoutItems* to the content of *MenuBarItem*, *MenuFlyout*, or *MenuFlyoutSubItem*.
+
+![Three radio menu flyout items in a View goup that allow a user to select the size of icons](RadioMenuFlyoutItems.png)
```

Comment:
There's a bug to update the selection visual from a check mark to a dot. I'll update the screenshot once that work is checked in.

Reply:
_Really_ good idea :)